src/main/webapp/display-event.html

```diff
@@ -48,6 +48,15 @@
             <span class="tag environment">environment</span>
           </div>
         </div>
+        <div class="footer">
+          <p>
+            Icons made by 
+            <a href="https://www.flaticon.com/authors/freepik"
+            title="Freepik"> Freepik</a> from 
+            <a href="https://www.flaticon.com/" title="Flaticon">
+            www.flaticon.com</a>
+          </p>
+        </div>
```

Comment:
Will this be added to the main footer later?

Reply:
Since I only used the icons on the event page, I didn't add it to the main footer on the other pages.